lib/routes/totp.js

```diff
@@ -64,7 +65,9 @@ module.exports = (log, db, mailer, customs, config) => {
           .then(createResponse)
           .then(() => reply(response), reply)
 
-        const secret = otplib.authenticator.generateSecret()
+        const authenticator = new otplib.authenticator.Authenticator()
```

Comment:
Not directly related to code window size, but changed this to be consistent with how we use `otplib.authenticator` in other files and repos.